src/test/custom_node_output_allocator_test.cpp

```diff
@@ -53,36 +55,31 @@ int NodeLibraryCheckingReleaseCalled::getOutputsInfo(struct CustomNodeTensorInfo
 
 int NodeLibraryCheckingReleaseCalled::release(void* ptr, void* customNodeLibraryInternalManager) {
     releaseBufferCalled = true;
-    return 4;
+    return 0;
```

Comment:
why 4=>0?

Reply:
because it then unnecessarily logs error in those tests. For this test it is correct to return 0 here since we only care that this is called (std::vector takes care of memory cleanup) and shouldn't create unnecessary log. This lead to unneeded investigation when rewriting this.